### PR TITLE
Input

### PR DESCRIPTION
**Pull request by Lokeshkhati**
image

image

image

Please review this input component

### CODE REVIEW

File: css/components/input.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    border-radius: 4px;
+    border: 1px solid var(--primary-color);
+    margin: 1rem;
+    width: 90%;
+  }
+  input:focus {
+    outline: none;
+  }
+  .signup-form-container {
+    width: 400px;
```
Comment (Tanjum786) [inline css/components/input.css]: Just a small suggestion use `rem` instead of `px` for a more responsive

**Review by Tanjum786 — APPROVED** [review]
do changes in unit taking. other than Everything looks good for me.

File: css/components/input.css (hunk 1 above)
Comment (Lokeshkhati) [reply css/components/input.css]: made the suggested  changes

**Comment by adithyark17** [comment]
Looks good and can be merged. @Tanjum786 suggestion helps

**Pull request merged**